themes/default/subtemplates/popover-full-size-img.inc.tpl

```diff
@@ -0,0 +1,5 @@
+ <template id="tmpl-img-popover">
```

Comment:
Seems superfluous and can be handled with the JavaScript code of the else part of the condition, see above.